src/components/Global/Navbar.js

```diff
@@ -25,7 +25,7 @@ const Navbar = () => {
             <div className="flex items-center justify-between w-full md:w-auto">
               <Link href="/">
                 <>
-                 <a href="https://webxdao.github.io/">
+                 <a href="/">
```

Comment:
```suggestion
                 <a>
```
In fact we don't even need the a href, because we use nextjs/Link :)

Reply:
@mkubdev This was added because of the bug. making the logo not working